### PR TITLE
create a launch config for flutter connect

### PR DESCRIPTION
**Pull request by devoncarew**
- create a launch config for flutter connect (fix https://github.com/dart-atom/dartlang/issues/1023)
- share the css between the jobs and connect dialog
- run the daemon connect discovery request in a job, in order to show progress to the user

@jason-simmons @danrubel

### CODE REVIEW

**Comment by jason-simmons** [comment]
lgtm

File: lib/flutter/flutter_daemon.dart [hunk 1]
```diff
@@ -465,3 +467,30 @@ class LogMessage {
...
+class DaemonRequestJob extends Job {
+  final PerformRequest _fn;
+
+  DaemonRequestJob(String name, this._fn) : super(toTitleCase(name));
+
+  bool get quiet => true;
+
+  Future run() {
+    return _fn().catchError((e) {
+      if (!analysisServer.isActive) return null;
```
Comment (danrubel) [inline lib/flutter/flutter_daemon.dart]: Sorry for my lack of knowledge here, but...
How is analysisServer.isActive related to a daemon request job?
Why does it suppress the notification?

**Comment by danrubel** [comment]
LGTM

File: lib/flutter/flutter_daemon.dart (hunk 1 above)
Comment (devoncarew) [inline lib/flutter/flutter_daemon.dart]: This is (poorly) copy-pasted code. Will fix!

**Pull request merged**